mediathread/assetmgr/api.py

```diff
@@ -147,7 +221,13 @@ def render_list(self, request, record_owner, record_viewer, assets, notes):
         ctx = {}
 
         # Sort notes with tastypie's built-in ordering
-        notes = self.apply_sorting(notes, request.GET.dict())
+        note_sort = request.GET.dict()
+
+        # SherdNote doesn't have selections to sort on
+        if re.match(r'^-?selections$', note_sort.get('order_by', '')):
+            note_sort['order_by'] = 'title'
+
+        notes = self.apply_sorting(notes, note_sort)
```

Comment:
The [AssetCollectionView](https://github.com/ccnmtl/mediathread/blob/master/mediathread/assetmgr/views.py#L1154) optimizes the render by paginating (slicing) the queryset before [it renders it out](https://github.com/ccnmtl/mediathread/blob/master/mediathread/assetmgr/views.py#L1159). (Rendering being expensive). Placing this sorting logic results in sorted pages, but not a full queryset sort. This logic needs to be moved up a level into the view.

Reply:
This part does seem to be working, sorting the annotations tied to each asset returned. I don't think it's necessary to sort the annotations here, so I'll just remove this part, for clarity.

Reply:
I see what you're talking about though - my asset sort is not working on the entire set of assets still. I'll fix that up with your tips.